### PR TITLE
get_index method reorganizing

### PR DESCRIPTION
**Pull request by Honcharov12**
Change get_index method and delete unused classes.

### CODE REVIEW

**Comment by jeanleonov** [comment]
Just to make it easier to read the PR:
this branch is based on https://github.com/AppScale/appscale/pull/2931 and https://github.com/AppScale/appscale/pull/2913, so those PRs should be merged first.

File: SearchService/solr_interface.py [hunk 1]
```diff
@@ -227,7 +226,7 @@ def update_document(self, app_id, doc, index_spec):
     solr_doc = self.to_solr_doc(doc)
 
     index = self.get_index(app_id, index_spec.namespace(), index_spec.name())
-    updates = self.compute_updates(index.name, index.schema.fields,
+    updates = self.compute_updates(index.name, index.schema,
```
Comment (jeanleonov) [inline SearchService/solr_interface.py]: `solr_doc.fields)` can be moved to this line now and it will fit 80 signs.

**Comment by jeanleonov** [comment]
As you're changing test files, could you also make changed fields fit 80 signs?
I also looked into build results - majority of failures are related to appscale and appscale-tools versions missmatch. Merge latest master to fix it.

**Comment by Honcharov12** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/5762/

**Pull request merged**